src/example/FluxOnly/FluxOnly.js

```diff
@@ -0,0 +1,139 @@
+import React from 'react';
+import {Store, Url, Actions} from '../..';
+import styles from './FluxOnly.css';
+
+const Switch = React.createClass({
+
+  getInitialState() {
+    return Store.getQuery();
+  },
+
+  componentDidMount() {
+    this.unsubscribe =
+      Store.subscribe(this.onChange);
+  },
+
+  componentWillUnmount() {
+    this.unsubscribe();
+  },
+
+  switchComponents() {
+    let query = 'Nothing';
+
+    if ('switch' in this.state) {
+      query = this.state.switch;
+    }
+    return query;
+  },
+
+  onChange() {
+    this.setState(Store.getQuery());
+  },
+
+  render() {
+    return (
+      <div className={styles.content}>
+        <p>Current state is: {this.switchComponents()} Clicked</p>
+      </div>
+    );
+  }
+});
+
+const FluxWithUrls = React.createClass({
+
+  render() {
+    return (
+      <div className={styles.fluxonly}>
+        <div>
+          <h2>Using component-router's Urls</h2>
+        </div>
+        <div>
+          <Url query={{switch: 'First'}}>
+            Render First Component
+          </Url>
+          <Url query={{switch: 'Second'}}>
+            Render Second Component
+          </Url>
+          <Switch />
+        </div>
+      </div>
+    );
+  }
+
+});
+
+const PureFlux = React.createClass({
+
+  getInitialState() {
+    return Store.getQuery();
+  },
+
+  componentDidMount() {
+    this.unsubscribe =
+      Store.subscribe(this.onChange);
+  },
+
+  componentWillUnmount() {
+    this.unsubscribe();
+  },
+
+  onChange() {
+    this.setState(Store.getQuery());
+  },
+
+  onClick(newVal) {
+    return event => {
+      event.preventDefault();
+      Store.dispatch(Actions.navigateTo({pathname: '/', query: newVal}));
+    };
+  },
+
+  activeClass({switch: val}) {
+    let cName = '';
+
+    if (this.state[`switch`] === val) {
```

Comment:
Theres obviously a *really, really* good reason, whatever it was 

![popkey](https://m.popkey.co/vf9iYtYmV3oHcZxFTBlDKp1EQQw=/original-d8d551f8-5747-4b10-b8c6-9b32d936b24e.gif/JNgE.gif?c=desktop&l=search&f=.gif)

...i'll change it